src/main/java/ir/nft/security/oauth2manager/controller/OAuth2ActivitiesController.java

```diff
@@ -0,0 +1,29 @@
+package ir.nft.security.oauth2manager.controller;
+
+import ir.nft.security.oauth2manager.dto.domain.OAuth2ActivityDTO;
+import ir.nft.security.oauth2manager.service.OAuth2ActivityService;
+import jakarta.validation.Valid;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("${app.base-api-url}" + "/activities")
```

Comment:
base uri is not a string that constantly change.
create class for rest uri constant with public static final sting instead of reading from properties file